Sources/SwiftKuery/Table.swift

```diff
@@ -159,19 +294,53 @@ open class Table: Buildable {
         return self
     }
     
-    /// Add a primary key to the table.
-    ///
-    /// - Parameter columns: Columns that constitute the primary key.
-    /// - Returns: A new instance of `Table`.
+    /**
+     Function to set a single `Column` instances` as a primary key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, the primary key is set to the `id` column for the table `personTable`.
+     ```swift
+     public class PersonTable: Table {
+        let tableName = "personTable"
+        let id = Column("id", Int32.self, notNull: true)
+        let firstName = Column("firstName", String.self, notNull: true)
+        let lastName = Column("lastName", String.self, notNull: true)
+     }
+     var personTable = PersonTable()
+     personTable = personTable.primaryKey(idColumn)
+     ```
+    
+     - Parameter columns: Single Column that constitute the primary key.
+     - Returns: A new instance of `Table`.
+    */
     public func primaryKey(_ columns: Column...) -> Self {
         return primaryKey(columns)
     }
     
-    /// Add a foreign key to the table.
-    ///
-    /// - Parameter columns: An Array of columns that constitute the foreign key.
-    /// - Parameter references: An Array of columns of the foreign table the foreign key references.
-    /// - Returns: A new instance of `Table`.
+    /**
+     Function to set multiple `Column` instances, as a composite foreign key, in the `Table` instance.
+     This function calls the composite primaryKey function with a single column to create a single primary key.
+     ### Usage Example: ###
+     In this example, columns for first and last name are initialized and a `Table` instance called personTable is created. A "personTable" foreign key is then set to be a composite of firstColumn and lastColumn, which reference columns describing the persons salary.
```

Comment:
The biggest problem with these foreign key example usage sections is as previously pointed out by Enrique - 

both these example for foreign key don't present what a foreign key is normally. A foreign key shouldn't be the id key of the table and it should normally reference the primary key of another table. So for example:
```
public class EmployeeTable: Table {
 let id = Column(...)
}
let work_id = Column(....)
personTable.foreignKey(work_id, references: [EmployeeTable.id])
```